htdocs/components/05_DataTable.js

```diff
@@ -118,8 +118,9 @@ Ensembl.DataTable = {
         $('.dataTables_info, .dataTables_paginate, .dataTables_bottom', tableSettings.nTableWrapper)[tableSettings._iDisplayLength === -1 ? 'hide' : 'show']();
         
         var data          = this.data();
-        var defaultHidden = data.defaultHiddenColumns || [];
-        var hiddenCols    = $.map(tableSettings.aoColumns, function (c, j) { return c.bVisible ^ defaultHidden[j] ? null : j * (defaultHidden[j] ? -1 : 1); }).join(',');
+        var hiddenCols = tableSettings.aoColumns.reduce(function (accumulator, column, index) {
```

Comment:
Array.reduce() method has no issues with the js minification program. (Thanks @azangru for testing)